tests/pom.xml

```diff
@@ -14,6 +14,7 @@
     <name>apicurio-registry-tests</name>
 
     <properties>
+        <groups></groups>
```

Comment:
?

Reply:
is needed for this https://github.com/Apicurio/apicurio-registry/pull/681/files#diff-5337bf8d5cd79ec462f8b8d2c5fe2d0bR259
and the value is set when using one of the profiles I added at the end of the file. Or do you mean it will work anyways without setting the empty property first?

Reply:
Oh I see.  Nevermind. :)